formResponses.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/kataras/iris"
+)
+
+// formIDs maps the query parameter values to the CCB form IDs
+// this is used in building the request to CCB for form responses
+var formIDs = map[string]string{
```

Comment:
Use `int` instead of `string`?

Reply:
Using a string since it's an ID and is not meant to be mutated.